core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -307,9 +307,10 @@ class UserService(protected val userInfo: UserInfo, val dataSource: SlickDataSou
     }.map(RequestComplete(_))
   }
 
-  def unregisterBillingProject(projectName: RawlsBillingProjectName): Future[PerRequestMessage] = {
+  def unregisterBillingProject(projectName: RawlsBillingProjectName, ownerInfo: Map[String, String]): Future[PerRequestMessage] = {
+    val ownerUserInfo = UserInfo(RawlsUserEmail(ownerInfo("newOwnerEmail")), OAuth2BearerToken(ownerInfo("newOwnerToken")), 3600, RawlsUserSubjectId("0"))
```

Comment:
do we not have the full userInfo passed in from gpalloc? seems gross (yet harmless) to put in a false subject id but....gross seems to be a theme with these endpoints

Reply:
nvm i see in workbench-libs that we construct this userInfo just from a token and an email

Reply:
passing the full userinfo was what got me into the mess of json format grossness. this was my extremely lazy sidestep